### DISCUSSION TITLE
Dynamic sorting (show users with matching city/country higher)

### DISCUSSION THREAD

**Discussion by benascbr** [meilisearch/product#664] [Feedback & Feature Proposal]
Hey there. Thank you for such wonderful tool.

Consider we have users table with 10mil rows.
To sort users we are using int `user.score` which takes into account many things. Example: last online date
As bonus feature we would like to implement one more dynamic score. If viewer and other user city matches add 100 to score. If country matches add 200 to score.
After reading docs I couldn't find such thing. Only thing was sorting based on distance but that wouldn't work for us.
Any help would be appreciated.

**Comment by macraig** [comment]
Hi @benascbr , thank you for reaching out! I've moved your discussion to our product repo as sounds like you have a feature request 😄 

I'd love to understand your use case better. Could you perhaps expand on why sorting via distance didn't work for your use case?
